examples/with-jest/index.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import glamorous from 'glamorous'
+import glamorous from '../../src'
```

Comment:
I don't know about this. I would prefer that the examples are self-contained and can be ripped out and still work. This way people can `cp -r examples/with-jest/ ~/my-new-project` and build something new.